release/models/gnsi/openconfig-gnsi-certz.yang

```diff
@@ -32,6 +32,14 @@ module openconfig-gnsi-certz {
 
   oc-ext:openconfig-version "0.5.0";
 
+  revision 2024-03- {
+    description
+      "align yang with the openconfig/gnsi version as of:
+      https://github.com/openconfig/gnsi/tree/e08ebfe537e1603015291898e6d5e9abe1562a0b
+      (update counter names)";
```

Comment:
I suggest copying the description of the revision(s) that introduced these same changes here.  This way we can think of the content of this module much more independently from the repository/storage system used.  

The distinction I'd like to make is content in these revision statements and the git log are relevant (IMO), but the  history of changes for openconfig/public repo should be hermetic/independent of other repositories)

Reply:
addressed in https://github.com/openconfig/public/pull/1061/commits/c0ffe5e385876828242f0eb6de7d9f6508d55143

That makes sense - I copied the description from the gNSI repo here: https://github.com/openconfig/gnsi/blob/718ee9102f4676e11485ff151e9caa4fa74f57e1/certz/gnsi-certz.yang#L30